app/qml/components/MMComboBoxDrawer.qml

```diff
@@ -0,0 +1,198 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import "../inputs"
+
+Drawer {
+  id: control
+
+  property alias title: title.text
+  property alias model: listView.model
+  property alias dropDownTitle: title.text
+  property bool multiSelect: false
+
+  padding: 20 * __dp
+
+  signal featureClicked( var selectedFeatures )
+
+  width: ApplicationWindow.window.width
+  height: (mainColumn.height > ApplicationWindow.window.height ? ApplicationWindow.window.height : mainColumn.height) - 20 * __dp
+  edge: Qt.BottomEdge
+
+  Rectangle {
+    color: roundedRect.color
+    anchors.top: parent.top
+    anchors.left: parent.left
+    anchors.right: parent.right
+    height: 2 * radius
+    anchors.topMargin: -radius
+    radius: 20 * __dp
+  }
+
+  Rectangle {
+    id: roundedRect
+
+    anchors.fill: parent
+    color: __style.whiteColor
+
+    Column {
+      id: mainColumn
+
+      width: parent.width
+      spacing: 20 * __dp
+      leftPadding: control.padding
+      rightPadding: control.padding
+      bottomPadding: control.padding
+
+      Row {
+        width: parent.width - 2 * control.padding
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        Item { width: closeButton.width; height: 1 }
+
+        Text {
+          id: title
+
+          anchors.verticalCenter: parent.verticalCenter
+          font: __style.t2
+          width: parent.width - closeButton.width * 2
+          color: __style.forestColor
+          horizontalAlignment: Text.AlignHCenter
+          verticalAlignment: Text.AlignVCenter
+          elide: Text.ElideRight
+        }
+
+        Image {
+          id: closeButton
+
+          source: __style.closeButtonIcon
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: control.visible = false
+          }
+        }
+      }
+
+      MMSearchEditor {
+        id: searchBar
+
+        width: parent.width - 2 * control.padding
+        placeholderText: qsTr("Text value")
+        bgColor: __style.lightGreenColor
+        visible: control.model.count > 4
```

Comment:
Can we have this as a property instead of some random constant? It should also be 5, not 4

```suggestion
        visible: control.model.count > internal.minFeaturesCountToFullScreenMode
```